### PR TITLE
Fan Page.

### PR DESCRIPTION
**Pull request by huskerman34**
Still working on the project. Having issues with the icon. It doesn't seem to work. I need to  adjust the layout of the webpage and the size of the fonts and jpegs.

### CODE REVIEW

**Comment by huskerman34** [comment]
Made some changes. Icon showed on my web page, Will add fonts to this

**Comment by huskerman34** [comment]
Updating

**Comment by huskerman34** [comment]
Should be updated

**Comment by huskerman34** [comment]
My git hub and home is a mess. I have files every where!!!! Please help!!!!

File: Neverwinter.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE HTML>
```
Comment (mbMosman) [inline Neverwinter.html]: This should be your index.html page, otherwise it will not be seen.

**Comment by mbMosman** [comment]
You should not have spaces in file names that will go to the web.  Something like "Neverwinter poster.jpg" needs to have the space removed.  Change it to an underscore or dash, or use camelCase.
Neverwinter-poster.jpg  or  Neverwinter_poster.jpg  or NeverwinterPoster.jpg.  

You also need to make a folder for your images and then use a path to that folder in your file names, like `images/Neverwinter-poster.jpg`

File: Neverwinter.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<ol type="A">
+<li>Swordcoast</li>
+<li>Icewind Dale</li>
+<li>Tiamat</li>
+</ol>
+<div id="content">
+<strong>Contact Edgar F Greenwood</strong>
+<a href="http://EdgarDrizzit.com">Huskerman34@hotmail.com</a>
+</div>   
+<link rel="stylesheet" href="style.css">
```
Comment (mbMosman) [inline Neverwinter.html]: The link tag should go inside the `<head>` section of your page, up near the top.

File: Neverwinter.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+<li>Tiamat</li>
+</ol>
+<div id="content">
+<strong>Contact Edgar F Greenwood</strong>
+<a href="http://EdgarDrizzit.com">Huskerman34@hotmail.com</a>
+</div>   
+<link rel="stylesheet" href="style.css">    
+
+<footer>
+<small><i>Copyright &copy; 2016 <span class=Edgarfgreenwood>Edgar F Greenwood</span></i></small>
```
Comment (mbMosman) [inline Neverwinter.html]: You shouldn't use tags like `<small>`, `<i>` or `<b>` in the HTML just to alter the display.  Use CSS instead.  Only use the HTML tags if there is a semantic reason to do so to put emphasis on a particular word.

You've also got a lot of images here that I suspect are not your own.  You need to credit the copyright/trademark holders of things like Neverwinter and credit the image owners (hopefully with permission to use their resources).

File: Neverwinter.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+<p>Birdsong Langer is a Sun Elf. He is skilled in the Hunter Ranger profession. He is a support fighter. He uses Cordon of Arrows and Foxes Cunning to his disposal.We will observe Birdsong and his companions through there trails and tribulatons as they venture to certain places like</p>
+<img src="neverwinter_nights_dragon_sky_mountain_wings_22041_3840x2160.png" height="600" width="800" alt="Dragon" >
+<ol type="A">
+<li>Swordcoast</li>
+<li>Icewind Dale</li>
+<li>Tiamat</li>
+</ol>
+<div id="content">
+<strong>Contact Edgar F Greenwood</strong>
+<a href="http://EdgarDrizzit.com">Huskerman34@hotmail.com</a>
```
Comment (mbMosman) [inline Neverwinter.html]: This needs to be a mailto link.  The email should be in the href and your name in between the open/close tags.

File: Neverwinter.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+   
+<p>Birdsong Langer is a Sun Elf. He is skilled in the Hunter Ranger profession. He is a support fighter. He uses Cordon of Arrows and Foxes Cunning to his disposal.We will observe Birdsong and his companions through there trails and tribulatons as they venture to certain places like</p>
+<img src="neverwinter_nights_dragon_sky_mountain_wings_22041_3840x2160.png" height="600" width="800" alt="Dragon" >
+<ol type="A">
+<li>Swordcoast</li>
+<li>Icewind Dale</li>
+<li>Tiamat</li>
+</ol>
+<div id="content">
+<strong>Contact Edgar F Greenwood</strong>
```
Comment (mbMosman) [inline Neverwinter.html]: You should use CSS to make this bold, not the `<strong>` tag.

File: Neverwinter.html [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+<html lang="en">
+<head>
+<title>Neverwitner Nights</title>
+<meta charset="UTF-8">
+<link rel="icon" href="favicon.ico" type="image/x-icon">
+</head>
+<body>
+<h1>Neverwinter</h1>
+ <nav>
+ <b> Swordcoast &nbsp; Tiamat &nbsp; Icewind Dale &nbsp;</b>
```
Comment (mbMosman) [inline Neverwinter.html]: You should use CSS to make these bold, not the `<b>` tag.

File: Neverwinter.html [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+ </nav> 
+<main>
+<h2>Why Neverwinter</h2>
+<p>Neverwinter nights is one of many stories from the world of Dungeon and Dragons. Im an avid fan of Dungeons and Dragons. I used to play the table top game and now I read books by Ed Greenwood and R.A Salvatore. I do play Dungeons and Dragons Online once in a while. Neverwinter gameplay is has more of an arcade feel as there are few hotkeys. The mechanics are great!!!! PVP is also fun. We will follow the trails of a Hunter Ranger, Birdsong Langer.  </p> 
+<h2>Birdsong Langer</h2>
+   
+<p>Birdsong Langer is a Sun Elf. He is skilled in the Hunter Ranger profession. He is a support fighter. He uses Cordon of Arrows and Foxes Cunning to his disposal.We will observe Birdsong and his companions through there trails and tribulatons as they venture to certain places like</p>
```
Comment (mbMosman) [inline Neverwinter.html]: There are some typo/grammatical errors in the text you might want to correct to show off your site and skills at it's best.

File: style.css [hunk 8]
```diff
@@ -0,0 +1,28 @@
+body {background-color:#AF670c; color: black; }
+
+#content {color: #ff00cc; } 
+
+.Edgarfgreenwood {color:aquamarine;}
+H1 { font-family: Lucida Handwriting; }
```
Comment (mbMosman) [inline style.css]: You should use a full font family with fallback fonts listed as shown in the textbook and web notes for CSS Font Stack:  https://htc-ccis1301.github.io/fall-2016/morea/graphics-text/read-graphics-text.html

Also, font names that are more than one word need to be in quotes, like "Lucida Handwriting".  This is a validation error.

File: Neverwinter.html [hunk 9]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+<title>Neverwitner Nights</title>
```
Comment (mbMosman) [inline Neverwinter.html]: typo

File: style.css [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+footer {font-style: italic; }
+h1 {text-align: center; }
+nav{text-align: center; }
+h1 {line-height: 2.0em; }
+h1{font-size:3.0em; }
+footer{font-size: .40em }
+p {font-family:fantasy; }
+title {font-style: oblique; }
```
Comment (mbMosman) [inline style.css]: You can't change the font on the title (that is what is show in the browser window/tab)

File: style.css [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+nav{font-size: 2.0em; }
+li {font-family:fantasy; font-size: 1.5em; }
+H1 {background-image: url(main-background-footer.png); background-repeat: no-repeat; background-position: center; }
+H2 {font-family: cursive; font-size:2.0em; }
+nav{font-style:italic; }
+H1 {color: #ff8000; }
+ol{list-style-type: upper-alpha;}
+a {font-weight: bold; font-size: 2.0em; }
+h1 {text-transform: uppercase; }
```
Comment (mbMosman) [inline style.css]: You should combine all of these style rules for the same tag into one rule for better maintainability and readability.

**Pull request closed without merging**